spring-boot-samples/spring-boot-sample-cache-guava/src/main/resources/application-spec.properties

```diff
@@ -0,0 +1,2 @@
+spring.cache.type=guava
+spring.cache.guava.spec=softValues
```

Comment:
I don't think any of the other sample goes that far. Users are free to checkout this sample and play with it by adding configuration key (or code to it). I'd stick with a simple `GuavaCacheApplication` and a `GuavaCacheApplicationTests`